src/uct/base/uct_vfs_attr.c

```diff
@@ -0,0 +1,32 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2021.  ALL RIGHTS RESERVED.
```

Comment:
plz use `Copyright (C) 2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.`, 
like in other new files

Reply:
Fixed